### ISSUE TITLE
Validation loss calculation

### ISSUE THREAD

**Issue by kulkarnikeerti** [yjh0410/PyTorch_YOLO-Family#20]
Hi @yjh0410 @HIT-k544 

I would like to know why the validation loss calculation is not done. Is it on purpose or you missed it? As per my understanding for model to learn the validation loss is important.

Thanks
Keerti

**Comment by yjh0410** [comment]
@kulkarnikeerti 
Sorry, I did not implement the function of calculating validation loss, because in my understanding, AP is the most important and most intuitive evaluation indicator. 

I think it should be easy to deploy the functionality to calculate validation loss into this project.

**Comment by kulkarnikeerti** [comment]
Okay, understood. Thanks @yjh0410
